java/vortex-spark/src/main/java/dev/vortex/spark/read/VortexScanBuilder.java

```diff
@@ -50,18 +55,29 @@ public VortexScanBuilder addAllPaths(Iterable<String> paths) {
     }
 
     public VortexScanBuilder addAllColumns(Iterable<Column> columns) {
-        this.columns.addAll(columns);
+        for (Column column : columns) {
+            this.columns.add(column);
+        }
```

Comment:
This change seems unnecessary; yeah?

Reply:
Unfortunately List.addAll takes a Collection, and an Iterable is not a Collection, so this is the way we need to do it. It is dumb.